src/network/api-crud.js

```diff
@@ -0,0 +1,29 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { post, get, fetchGet, fetchPost } from './network-utils';
+import { ADD_TODO, ALL_USERS, LOGIN, USER_RECORDS } from '../constants/api-constants';
+
+export const addNewTodo = async (newRecord, callback) => {
+    fetchPost(ADD_TODO, {
+        todo: newRecord.todo,
+        completed: false,
+        userId: newRecord.userId,
+    }).then((res) => {
```

Comment:
use async await istead of .then and wrap it in try catch to handle error

Reply:
done